tests/integration/test_charm.py

```diff
@@ -144,7 +144,7 @@ async def test_build_and_deploy(ops_test: OpsTest, db_driver, use_router) -> Non
             apps=[DB_CHARM[db_driver]["app_name"]],
             status="active",
             raise_on_blocked=True,
-            timeout=15 * 60,
+            timeout=20 * 60,
```

Comment:
@dragomirp any particular reason to increase this value?

Reply:
Advanced sleep() programming! ;-)

Reply:
The one of the postgresql tests timed out [twice](https://github.com/canonical/sysbench-operator/actions/runs/8645051269/job/23702497568) in a row. The timeout used in postgresql VM tests is usually 25 mins.